README.md

```diff
@@ -7,7 +7,8 @@ and Linux. Please visit [xqemu.com](http://xqemu.com) to learn more.
 Build Status
 ------------
 
-| Build   | Windows                                                                                                                                                               | macOS                                                                                                                                | Linux                                                                                                                                |
-| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------ |
-| Release | [![Build status](https://ci.appveyor.com/api/projects/status/i1m0keigjabxg170/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu/branch/master) | [![Build status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/4)](https://travis-ci.org/xqemu/xqemu) | [![Build status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/2)](https://travis-ci.org/xqemu/xqemu) |
-| Debug   | [![Build status](https://ci.appveyor.com/api/projects/status/i1m0keigjabxg170/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu/branch/master) | [![Build status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/3)](https://travis-ci.org/xqemu/xqemu) | [![Build status](https://travis-matrix-badges.herokuapp.com/repos/xqemu/xqemu/branches/master/1)](https://travis-ci.org/xqemu/xqemu) |
+| Platform | Build Status |
+|----------|--------------|
+| Windows | [![Build status](https://github.com/xqemu/xqemu/workflows/Build%20(Windows)/badge.svg?branch=master)](https://github.com/xqemu/xqemu/actions?query=branch%3Amaster) |
```

Comment:
Is there any repo where I can see these badges? I checked mborgerson/xqemu but they weren't there.
I also would like to see if there are per-platform links that we can use (nit).

Reply:
No, the badges won't work here because they have the xqemu URL. They look like the old badges for the most part

Reply:
Nevermind, they are here: https://github.com/mborgerson/xqemu/workflows/Build%20(macOS)/badge.svg - this should work fine.